<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_DadosCliente</class>
 <widget class="QWidget" name="frm_DadosCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #eaf2f8;
    border-radius: 8px;
}</string>
  </property>
  <widget class="QLabel" name="lbl_nome">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nome">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_celular">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Celular: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cpf">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>CPF:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffebee; 
    border: 2px solid #ffcdd2;
    border-radius: 10px;
    color: #b71c1c; 
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/icon_cancelar/cancelar.png);
    background-repeat: no-repeat;
    background-position: center;
    padding-left: 40px;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #ffcdd2;
    border-color: #e57373;
    color: #d32f2f; 
}

QPushButton:pressed {
    background-color: #e57373;
    border-color: #b71c1c;
    padding-left: 44px;
    padding-top: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_celular">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>(00) 0 0000-0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cpf">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>000.000.000-00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Rua">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Rua:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Rua">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cidade">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cadastrar_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 2px solid #d1c4b2;
    border-radius: 12px;
    color: #5a5a5a;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/icon_cadastrar/cadastrar.png); 
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #f2ebe0;
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf; 
    border-color: #b39b8d; 
    padding-left: 12px; 
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cidade">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bairro">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Bairro:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_bairro">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cep">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>CEP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cep">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>00000-000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Numero">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Numero">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Número:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cidade_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_email">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon_cadastrar.qrc"/>
  <include location="icon_cancelar.qrc"/>
 </resources>
 <connections/>
</ui>
